src/webui/webapplication.cpp

```diff
@@ -49,101 +52,21 @@
-const QString SCOPE_THEME = "theme";
-const QString DEFAULT_ACTION = "index";
-const QString WEBUI_ACTION = "webui";
-const QString VERSION_INFO = "version";
-const QString MAX_AGE_MONTH = "public, max-age=2592000";
-
-#define ADD_ACTION(scope, action) actions[#scope][#action] = &WebApplication::action_##scope##_##action
-
-QMap<QString, QMap<QString, WebApplication::Action>> WebApplication::initializeActions()
-{
-    QMap<QString,QMap<QString, WebApplication::Action>> actions;
-
-    ADD_ACTION(public, webui);
-    ADD_ACTION(public, index);
-    ADD_ACTION(public, login);
-    ADD_ACTION(public, logout);
-    ADD_ACTION(public, theme);
-    ADD_ACTION(public, images);
-    ADD_ACTION(query, torrents);
-    ADD_ACTION(query, preferences);
-    ADD_ACTION(query, transferInfo);
-    ADD_ACTION(query, propertiesGeneral);
-    ADD_ACTION(query, propertiesTrackers);
-    ADD_ACTION(query, propertiesWebSeeds);
-    ADD_ACTION(query, propertiesFiles);
-    ADD_ACTION(query, getLog);
-    ADD_ACTION(query, getPeerLog);
-    ADD_ACTION(query, getPieceHashes);
-    ADD_ACTION(query, getPieceStates);
-    ADD_ACTION(sync, maindata);
-    ADD_ACTION(sync, torrent_peers);
-    ADD_ACTION(command, shutdown);
-    ADD_ACTION(command, download);
-    ADD_ACTION(command, upload);
-    ADD_ACTION(command, addTrackers);
-    ADD_ACTION(command, resumeAll);
-    ADD_ACTION(command, pauseAll);
-    ADD_ACTION(command, resume);
-    ADD_ACTION(command, pause);
-    ADD_ACTION(command, setPreferences);
-    ADD_ACTION(command, setFilePrio);
-    ADD_ACTION(command, getGlobalUpLimit);
-    ADD_ACTION(command, getGlobalDlLimit);
-    ADD_ACTION(command, setGlobalUpLimit);
-    ADD_ACTION(command, setGlobalDlLimit);
-    ADD_ACTION(command, getTorrentsUpLimit);
-    ADD_ACTION(command, getTorrentsDlLimit);
-    ADD_ACTION(command, setTorrentsUpLimit);
-    ADD_ACTION(command, setTorrentsDlLimit);
-    ADD_ACTION(command, alternativeSpeedLimitsEnabled);
-    ADD_ACTION(command, toggleAlternativeSpeedLimits);
-    ADD_ACTION(command, toggleSequentialDownload);
-    ADD_ACTION(command, toggleFirstLastPiecePrio);
-    ADD_ACTION(command, setSuperSeeding);
-    ADD_ACTION(command, setForceStart);
-    ADD_ACTION(command, delete);
-    ADD_ACTION(command, deletePerm);
-    ADD_ACTION(command, increasePrio);
-    ADD_ACTION(command, decreasePrio);
-    ADD_ACTION(command, topPrio);
-    ADD_ACTION(command, bottomPrio);
-    ADD_ACTION(command, setLocation);
-    ADD_ACTION(command, rename);
-    ADD_ACTION(command, setAutoTMM);
-    ADD_ACTION(command, recheck);
-    ADD_ACTION(command, setCategory);
-    ADD_ACTION(command, addCategory);
-    ADD_ACTION(command, removeCategories);
-    ADD_ACTION(command, getSavePath);
-    ADD_ACTION(version, api);
-    ADD_ACTION(version, api_min);
-    ADD_ACTION(version, qbittorrent);
-
-    return actions;
-}
+constexpr Utils::Version<int, 3, 2> API_VERSION {2, 0, 0};
+
+const QString PATH_PREFIX_API {"/api/"};
```

Comment:
personally I prefer using `=` for string...

Reply:
I would prefer leave it as it is now.